src/server.ts

```diff
@@ -48,40 +58,20 @@ export function makeServer(appState: AppState): Express {
     next(new ServerError(404, 'not found'))
   })
 
-  // Client Error Route
-  app.use((err: Error, _req: Request, res: Response, next: NextFunction) => {
-    if (!(err instanceof ServerError)) {
-      return next(err)
-    }
-
+  // Error Route Handler
+  app.use((err: any, _req: Request, res: Response, _next: NextFunction) => {
+    const message = err.message ?? 'Internal Server Error'
+    const statusCode =
+      err instanceof ServerError ? err.status : err.statusCode ?? 500
     const response: ServerErrorResponse = {
       success: false,
-      message: err.message,
+      message: message,
```

Comment:
You could replace this with just `message` if you wanted.

Reply:
Fixed!